RadClass/H0.py

```diff
@@ -79,13 +106,19 @@ def run_channels(self, data, timestamp):
             nvec = self.x1 + self.x2
             p = 0.5
             for i, (x1, n) in enumerate(zip(self.x1, nvec)):
-                pval = stats.binom_test(x1, n, p,
-                                        alternative='two-sided')
+                # scipy.stats.binomtest will fail if n (# of trials)
+                # is less than 1 (possible for high-energy bins)
+                if int(n) < 1:
+                    pval = 1.0
+                else:
+                    pval = stats.binomtest(int(x1), int(n), p,
+                                           alternative='two-sided').pvalue
```

Comment:
note that `scipy.stats.binomtest` requires n>=1. This makes sense since you can't do a binomial test on 0 trials, which is possible if your channel or gross count-rate is very low (~0).